### ISSUE TITLE
Plugin is not compatible with Android Studio 3.1

### ISSUE THREAD

**Issue by danielesegato** [KeepSafe/dexcount-gradle-plugin#234]
Upgrading to Android Studio 3.1 the Android Plugin is updated to:
```
classpath 'com.android.tools.build:gradle:3.1.0-alpha01'
```

and the error

> Unable to load class 'com.android.builder.Version'. Possible causes for this unexpected error include:
> 
> - Gradle's dependency cache may be corrupt (this sometimes occurs after a network connection timeout.) Re-download dependencies and sync project (requires network)
> - The state of a Gradle build process (daemon) may be corrupt. Stopping all Gradle daemons may solve this problem. Stop Gradle build processes (requires restart)
> 
> Your project may be using a third-party plugin which is not compatible with the other plugins in the project or the version of Gradle requested by the project. In the case of corrupt Gradle processes, you can also try closing the IDE and then killing all Java processes.

is shown when try to sync gradle.

Removing dexcount plugin fix the issue.

**Comment by benjamin-bader** [comment]
Hi, I've pushed a snapshot build (version `0.8.2-SNAPSHOT`); if you're willing, would you please try it out?  Snapshots are in the Sonatype Snapshots maven repo; see our README for instructions.

Let me know if it works out for you!

**Comment by danielesegato** [comment]
Hi,

I've tried the snapshot, I can confirm it works again.

But I'm using

```groovy
classpath 'com.android.tools.build:gradle:3.1.0-alpha04'
```
and `gradle 4.4-rc4` just released.

cause i upgraded Android Studio in the meanwhile to _3.1 Canary 4_

I can confirm, anyway, the same issue reported for `alpha01` was present in `alpha04`.

**Comment by benjamin-bader** [comment]
Thanks!  Just so I'm clear - are you saying that the snapshot works for `alpha01` but _not_ `alpha04`?  Or does it work, generally?

**Comment by danielesegato** [comment]
No problem :)

I'm saying i tested it in `alpha04`, and I see no reason why it shouldn't work on `alpha01` as well, since they had the same error :)

**Comment by benjamin-bader** [comment]
Awesome!  In that case, I've shipped 0.8.2 as a proper release.  It should be visible on Maven Central within 20 minutes.

**Comment by danielesegato** [comment]
That's great! thank you!
